api/projects/submit.go

```diff
@@ -81,7 +81,18 @@ func submit(c echo.Context) error {
 	// TODO wait until container is complete
 	cli.ContainerWait(context.Background(), createResponse.ID)
 
-	return c.NoContent(http.StatusAccepted)
+	logsOptions := types.ContainerLogsOptions{
```

Comment:
we're gonna need to split these in the future, I guess we'll worry about it then. this is still valuable.